### PR TITLE
highlight late submissions

### PR DESCRIPTION
**Pull request by eladeyal-intel**
1. Add ORANGE tag if event was submitted after cfp end; or RED tag if submitted after hard deadline. The tag appears in event lists in a short form, and in an explicit form in the event details page.

2. Change creation date display to conference timezone.

see #490

image

image

### CODE REVIEW

File: config/locales/fr.yml [hunk 1]
```diff
@@ -1032,6 +1032,13 @@ fr:
     events_in_other_conferences: Évènements dans d’autres conférences
     feedback_not_enabled: Le système de retours n’est pas activé.
     hide_all: Tout cacher
+    indicator:
+      full:
+        submitted_after_hard_deadline: Soumis après la date limite
+        submitted_after_soft_deadline: Soumis après la date limite
+      short:
+        submitted_after_hard_deadline: En retard
+        submitted_after_soft_deadline: EN RETARD
```
Comment (erdgeist) [inline config/locales/fr.yml]: These do not look right.

File: config/locales/pt-BR.yml [hunk 2]
```diff
@@ -1019,6 +1019,13 @@ pt-BR:
     events_in_other_conferences: Eventos em outras conferências
     feedback_not_enabled: O sistema de feedback não está ativado.
     hide_all: esconda tudo
+    indicator:
+      full:
+        submitted_after_hard_deadline: Enviado após prazo final
+        submitted_after_soft_deadline: Enviado após o prazo
+      short:
+        submitted_after_hard_deadline: Atrasado
+        submitted_after_soft_deadline: ATRASADO
```
Comment (erdgeist) [inline config/locales/pt-BR.yml]: These neither ;)

File: config/locales/de.yml [hunk 3]
```diff
@@ -1010,6 +1010,13 @@ de:
     events_in_other_conferences: Events in anderen Konferenzen
     feedback_not_enabled: Das Feedback-System ist nicht aktiviert.
     hide_all: Alles verbergen
+    indicator:
+      full:
+        submitted_after_hard_deadline: Eingereicht nach harter Frist
+        submitted_after_soft_deadline: Eingereicht nach Fristablauf
```
Comment (erdgeist) [inline config/locales/de.yml]: from config/locales/de.yml, in German that would be "end_date: Enddatum
        hard_deadline: Harte Einreichungsfrist"

And I guess, these translations break conventions in other languages, as well.

Comment (eladeyal-intel) [reply config/locales/de.yml]: Your feedback is not clear enough for me. The "indicator full" is the string going for the event page, and the "indicator short" is the text going to the list pages. I thought just writing "Late" is enough becasue I did not want it to be too verbose

Comment (erdgeist) [reply config/locales/de.yml]: Sure, the short version is okay. But the long version of the term "hard deadline" and "soft deadline" already has had other translations. And now these mismatch. I showed you with the German example. Other translations don't even show differences between the two translations.

**Comment by eladeyal-intel** [comment]
I moved the tag to the "created" column; so now it shows only on `All Events` and `My events` but not on `attachmetns` tab or `event ratings` tab

image

image

image

File: config/locales/de.yml (hunk 3 above)
Comment (eladeyal-intel) [reply config/locales/de.yml]: So its

submitted_after_soft_deadline: Eingereicht nach Enddatum
submitted_after_hard_deadline: Eingereicht nach Harte Einreichungsfrist

?

**Pull request merged**